plugins/sqlalchemy/setup.py

```diff
@@ -0,0 +1,35 @@
+from setuptools import setup
+
+PLUGIN_NAME = "sqlalchemy"
+
+microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
+
+plugin_requires = ["flytekit>=0.17.0,<1.0.0", "sqlalchemy>=1.4.7"]
+
+__version__ = "0.0.0+develop"
+
+setup(
+    name=microlib_name,
+    version=__version__,
+    author="dolthub",
+    author_email="max@dolthub.com",
+    description="SQLAlchemy plugin for flytekit",
+    namespace_packages=["flytekitplugins"],
+    packages=[f"flytekitplugins.{PLUGIN_NAME}"],
+    install_requires=plugin_requires,
+    license="apache2",
+    python_requires=">=3.7",
+    classifiers=[
+        "Intended Audience :: Science/Research",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Artificial Intelligence",
+        "Topic :: Software Development",
+        "Topic :: Software Development :: Libraries",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    scripts=["scripts/flytekit_install_sqlalchemy.sh"],
```

Comment:
okay one last comment sorry.  this script is missing.  do we need this?  Can we move the dolt install script to the example in the future as well and just remove it, or is there a new script that's needed?

Reply:
no script needed, good catch i'll fix in a couple